src/components/Accordion/useAccordionContext.js

```diff
@@ -0,0 +1,11 @@
+import { createContext, useContext } from 'react'
+export const AccordionContext = createContext({
+  focusRef: {},
+  selected: [null],
+  toggleChildIsTarget: false,
+  expandedAll: [],
+  onToggle: undefined,
+  onNavigation: () => undefined,
+  id: null,
+})
+export const useAccordionContext = () => useContext(AccordionContext)
```

Comment:
Most of the documentation on context that I was reading (https://reactjs.org/docs/context.html) described context before hooks were introduced and I was briefly confused. 😂 

This looks great to me!